obspy/io/rg16/core.py

```diff
@@ -1,233 +1,845 @@
-# -*- coding: utf-8 -*-
-from __future__ import (absolute_import, division, print_function,
```

Comment:
Unfortunately we need to keep these lines until we drop python 3 support. They should be at the top (after the docstring) of each file.